app/models/manageiq/providers/embedded_ansible/automation_manager/playbook_runner.rb

```diff
@@ -1,20 +1,5 @@
 class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::PlaybookRunner < ::Job
-  DEFAULT_EXECUTION_TTL = 10 # minutes
-
-  # options are job table columns, including options column which is the playbook context info
-  def self.create_job(options)
-    super(name, options.with_indifferent_access)
-  end
-
-  def minimize_indirect
-    @minimize_indirect = true if @minimize_indirect.nil?
-    @minimize_indirect
-  end
-
-  def current_job_timeout(_timeout_adjustment = 1)
-    @execution_ttl ||=
-      (options[:execution_ttl].present? ? options[:execution_ttl].try(:to_i) : DEFAULT_EXECUTION_TTL) * 60
-  end
+  include AnsibleRunnerMixin
```

Comment:
@lfu Should this mixin be named differently since there is already something called an Ansible Runner https://github.com/ManageIQ/manageiq/blob/master/lib/ansible/runner.rb

Reply:
`AnsibleMiqJobMixin`?

Reply:
@lfu @bzwei 
What is the functionality overlap with the AnsibleRunner are they completely different.
We had some discussion in this PR https://github.com/ManageIQ/manageiq/pull/17759 when it was created. If they are both doing the same thing except for one we need a JobTemplate whereas the other one runs a playbook directly without the need for a Tower. We might want to call this AnsibleTowerJobRunner or AWXJobRunner. If the AnsibleRunner can launch playbooks without creating a Job template first we might have to change our implementation at some point.

Reply:
Current `PlaybookRunner` may not go away until we remove embedded tower and switch to native Ansible Runner. If we have concern not sharing the implementation in the future, we do not need this refactoring by extracting a mixin. Simply create a JobTemplateRunner class with some duplicate code.